provider/schema/execution.go

```diff
@@ -127,7 +170,7 @@ func (e ExecutionData) callTableResolve(ctx context.Context, client ClientMeta,
 	// check if channel iteration stopped because of resolver failure
 	if resolverErr != nil {
 		client.Logger().Error("received resolve resources error", "table", e.Table.Name, "error", resolverErr)
```

Comment:
I disagree. I think partial fetching shouldn't change the severity of log messages since a log message severity is an indicator of the issue severity (like the name says). Partial fetching doesn't solve or reduces the severity of issues but instead it only skips certain issues to make sure we that is get the most out of the run.

Reply:
I agree, but, wouldn't that be confusing log wise, its a so called "good red" in your log, when people see errors its usually alarming.